Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build stage
+FROM python:3.9-slim AS build

```

Comment:
The alias 'build' is correctly used for the build stage. However, ensure that the syntax 'AS build' is supported by your Docker version.